notification/app/notification/services/azure/WNSPushConverter.scala

```diff
@@ -82,4 +83,10 @@ class WNSPushConverter(conf: Configuration) {
     addedTime = gan.addedTime,
     debug = conf.debug
   )
+
+  private def toElectionResult(el: ElectionNotification) = wns.ElectionNotification(
```

Comment:
If this isn't already done, I think it would be worth checking with the win10 team to see if this isn't going to cause any issue.